Android/app/src/main/res/values/strings.xml

```diff
@@ -157,4 +157,5 @@
     <string name="sign_out">Sign out</string>
 
     <string name="coming_soon">Coming Soon!</string>
+    <string name="enter_start_date">Enter start date</string>
```

Comment:
We already have this string `prompt_start_date` in `strings.xml`

Also, remove string with key `prompt_end_date`